code/modules/mob/living/simple_animal/abnormality/teth/crumbling_armor.dm

```diff
@@ -120,6 +138,71 @@
 			to_chat(user, "<span class='userdanger'>A strange power flows through you!</span>")
 	return
 
+/mob/living/simple_animal/hostile/abnormality/crumbling_armor/ZeroQliphoth(mob/living/carbon/human/user)
+	MeltdownEffect()
+	datum_reference.qliphoth_change(3)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/crumbling_armor/proc/MeltdownEffect(mob/living/carbon/human/user)
+	var/list/potentialmarked = list()
+	var/list/marked = list()
+
+	for(var/mob/living/carbon/human/L in GLOB.player_list)
+		if(L.stat >= HARD_CRIT || L.sanity_lost || z != L.z) // Dead or in hard crit, insane, or on a different Z level.
+			continue
+		potentialmarked += L
+		to_chat(L, "<span class='danger'>You feel an overwhelming sense of foreboding...</span>")
+
+	SLEEP_CHECK_DEATH(10 SECONDS)
+	for(var/i=numbermarked, i>=1, i--)
+		var/mob/living/Y = pick(potentialmarked)
+		if(faction_check_mob(Y, FALSE) || Y.z != z || Y.stat == DEAD)
+			continue
+		if(Y in marked)
+			continue
+		marked+=Y
+		to_chat(Y, "<span class='userdanger'>Show me that you can stand your ground!</span>")
+		playsound(get_turf(Y), 'sound/abnormalities/crumbling/warning.ogg', 50, 0, 2)
+		Y.apply_status_effect(STATUS_EFFECT_COWARDICE)
```

Comment:
Add a visual indicator and delay before effect is actually applied/can punish the victim.